backend/isoko-backend/src/handlers/business/get-business-page.js

```diff
@@ -9,10 +13,32 @@ exports.getBusinessPageHandler = async (event) => {
    }
 
    console.info('received:', event);
+   const { businessId } = event.pathParameters; 
+
+   if (businessId == null) {
+      throw new Error(
+         `Missing query parameter 'businessId'. Request URL format: GET/business/{businessId}`
+      );
+   }
+
+   const queryParams = {
```

Comment:
nit: Can we rename this to `params` or something similar, queryParams is a bit confusing here